client/src/components/NavBar/NotificationRequest/NotificationRequest.tsx

```diff
@@ -24,11 +24,11 @@ const NotificationRequest: React.FC<IProps> = ({
 
   return (
     <Box display="flex" alignItems="center" m={2}>
-      <Avatar variant="square" src={`/${image}`} className={classes.large} />
+      <Avatar variant="square" src={`/${senderProfile.image}`} className={classes.large} />
       <Box>
         <Link onClick={changeUnreadToRead} to={type === 'Service' ? '/my-sitters' : '/messages'}>
```

Comment:
Per my previous comment, please match type to the allowed values based on the Notification model: ["serviceRequest", "serviceAvailable", "message"] :)